src/pymor/parameters/base.py

```diff
@@ -309,10 +361,11 @@ def __eq__(self, mu):
         return self.keys() == mu.keys() and all(np.array_equal(v, mu[k]) for k, v in self.items())
 
     def __str__(self):
-        return '{' + ', '.join(f'{k}: {format_array(v)}' for k, v in self.items()) + '}'
+        return '{' + ', '.join(f'{k}{"(t)" if self.is_time_dependent(k) else ""}: {format_array(v)}'
+                               for k, v in self.items()) + '}'
```

Comment:
Looking at the logging output of your example code: I don't quit like this way of printout for the new functionality. In the example code you provided, this will printout the value of the parameter in `t=0` without saying it. So I would rather either print the whole expression instead of mu(t=0) or have `diffusion(t = 0)` instead of `diffusion(t)` in the printout.